lethal-squirrels/Bangazon-Site
# Chaz remove order items

## REFERENCE

Ticket: #6 

## DESCRIPTION
When a user is looking at their shopping cart - when they click on `Remove Item` it will be removed from their shopping cart. 

## STEPS TO TEST
1. Log in
1. Navigate to shopping cart
1. Remove items - quantity numbers should go down
1. Try adding more products, watch them show up in the shopping cart and then remove them. Those items were never worthy of your hard earned money anyway. (Stop spending your money on cheap shit made in China. Buy American. USA USA USA)
